lib/timeline/Core.js

```diff
@@ -932,7 +940,7 @@ class Core {
     dom.root.style.height = util.option.asSize(options.height, `${autoHeight}px`);
 
     // calculate heights of the content panels
-    props.root.height = dom.root.offsetHeight;
```

Comment:
Apparently this is the row that is causing glichiness in your PR. 
Return this to what was before and we can merge this PR already!